JASP-Desktop/widgets/boundqmljagstextarea.cpp

```diff
@@ -92,17 +100,20 @@ void BoundQMLJAGSTextArea::checkSyntax()
 	// google: jags_user_manual (4.3.0) for documentation on JAGS symbols
 
 	// get the column names of the data set
-	std::vector<std::string> colnms = DataSetPackage::pkg()->getColumnNames();
-	std::set<std::string> columnNames(std::make_move_iterator(colnms.begin()), std::make_move_iterator(colnms.end()));
+	std::set<std::string> columnNames;
+	_textEncoded = tq(ColumnEncoder::columnEncoder()->encodeRScript(stringUtils::stripRComments(fq(_text)), &columnNames));
+	_usedColumnNames.clear();
```

Comment:
Why not just change _usedColumnNames to `std::set<std::string>` and using it directly?

Reply:
(If you want too, it is a blocker or anythin)

Reply:
ok I changed it to `std::set<std::string>`